app/presenters/addresses_presenter.rb

```diff
@@ -0,0 +1,11 @@
+class AddressesPresenter < Jsonite
+  property :attention
+  property :address_1
+  property :address_2
+  property :address_3
+  property :address_4
+  property :city
+  property :state_province
+  property :postal_code
+  property :country
+end
```

Comment:
Let's get some new lines at the end of these files. Your editor can probably be configured to add them on save.